jboss-cert-ldap-login-module/src/main/java/org/esbtools/auth/jboss/CertLdapLoginModule.java

```diff
@@ -63,6 +63,7 @@
     public static final String ROLES_CACHE_EXPIRY_MS = "rolesCacheExpiryMS";
     public static final String ENVIRONMENT = "environment";
     public static final String ALL_ACCESS_OU = "allAccessOu";
+    public static final String RETRY_INTERVAL_SECONDS = "retryIntervalSeconds";
 
     private static final String[] ALL_VALID_OPTIONS = {
             AUTH_ROLE_NAME, SERVER, PORT, SEARCH_BASE, BIND_DN, BIND_PWD, USE_SSL,
```

Comment:
You need to add RETRY_INTERVAL_SECONDS to ALL_VALID_OPTIONS, otherwise we'll get warnings in the logs